docs/pages/api/index.md

```diff
@@ -12,33 +12,115 @@ env = gym.make('CartPole-v0')
+
+    if done:
+      observation, info = env.reset(return_info=True)
+
 env.close()
 ```
 
 The output should look something like this
 
-![cartpole-no-reset](https://user-images.githubusercontent.com/28860173/129241283-70069f7c-453d-4670-a226-854203bd8a1b.gif)
+![lunar_lander](https://user-images.githubusercontent.com/15806078/153222406-af5ce6f0-4696-4a24-a683-46ad4939170c.gif)
+
+
+## Commonly used methods
+>### Stepping
+> `step(action: ActType) -> Tuple[ObsType, float, bool, dict]`
+> 
+>Run one timestep of the environment's dynamics. When end of episode is reached, you are responsible for calling `reset()`
+to reset this environment's state. Accepts an action and returns a tuple `(observation, reward, done, info)`.
+>
+>>**Parameters**
+>>- `action`(**object**): an action provided by the agent. This will oftentimes be a numpy array, but might also be 
+>> an integer (for discrete action spaces) or a more complex object.
+> 
+>>**Returns**
+>>
+>> This method returns a tuple `(observation, reward, done, info)`:
+>>- `observation` (**object**): agent's observation of the current environment.
+>> This may, for instance, be a numpy array containing the positions and velocities of certain objects.
+>>- `reward` (**float**) : amount of reward returned after previous action
+>>- `done` (**bool**): whether the episode has ended, in which case further `step()` calls will return undefined results.
+>> A done signal may be emitted for different reasons: Maybe the task underlying the environment was solved successfully, 
+>> a certain timelimit was exceeded, or the physics simulation has entered an invalid state. `info` may contain additional
+>> information regarding the reasone for a done signal.
+>>- `info` (**dict**): contains auxiliary diagnostic information (helpful for debugging, and sometimes learning)
+
+>### Resetting
+>`reset(*, seed: Optional[int] = None, return_info: bool = False, options: Optional[dict] = None,) -> Union[ObsType, tuple[ObsType, dict]]`
+>
+>Resets the environment to an initial state and returns an initial observation.
+>>**Parameters**
+>>- `seed`(**int** or **None**): The seed that is used to initialize the environments PRNG. If the environment does not already
+>> have a PRNG and `seed=None`(the default option) is passed, a seed will be chosen from some source of entropy (e.g. timestamp 
+or /dev/urandom). However, if the environment already has a PRNG and `seed=None` is pased, the PRNG will *not* be reset. 
+If you pass an integer, the PRNG will be reset even if it already exists. Usually, you want to pass an integer *right after the environment
+has been initialized and then never again*. Please refer to the minimal example above to see this paradigm in action.
+>>- `return_info`(**bool**): If true, return additional information along with initial observation. This info should be analogous
+to the info returned in `step`
+>>- `options`(**dict** or **None**):
```

Comment:
[...]: Additional information to specify how the environment is reset (optional, depending on the specific environment)

(feel free to rephrase this)